modules/eks/variables.tf

```diff
@@ -230,3 +231,13 @@ variable "workers_role_name" {
   type        = string
   default     = ""
 }
+
+variable "global_eks_nodepool_default_ami_type" {
```

Comment:
let's call this `node_pools_ami_type` or `node_pools_global_ami_type` if you prefer. It is large enough already and `eks` and `default` are redundant, also, we already have some variables called `node_pools_*`.

Reply:
`node_pools_global_ami_type` sounds good to me

Reply:
node_pools_global_ami_type was implemented